src/main/java/internetShop/mvc/ProductController.java

```diff
@@ -0,0 +1,48 @@
+package internetShop.mvc;
+
+import internetShop.entity.product.Product;
+import internetShop.service.ProductService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.util.List;
+
+@Controller
+public class ProductController {
+
+    @Autowired
+    private ProductService productService;
+
+    public void setProductService(ProductService productService) {
+        this.productService = productService;
+    }
+
+    @GetMapping("/")
+    public String main(Model model) {
+        List<Product> products = productService.getProducts();
+        model.addAttribute("products", products);
+        return "/products";
+    }
+
+    @GetMapping("/deleteProduct/{id}")
+    public String deleteProduct(
+            @PathVariable Long id
+    ) {
+        productService.deleteProduct(id);
+        return "redirect:/";
+    }
+
+    @PostMapping("/addProduct")
+    public String add(@RequestParam String name, Double price) {
```

Comment:
as well you could use such thing here 
instead @RequestParam name/price
Product product